project/app/components/Footer.js

```diff
@@ -7,7 +7,7 @@ const Footer = ({ toggleFlippable }) => (
   <p className="text-center mt-3">
     Made with <Icon title="Click to See Animated Card" name="heart" color="danger" onClick={toggleFlippable} />{' '}
     by{' '}
-    <Link url="https://www.availity.com" target="_blank">
+    <Link href="https://www.availity.com" url="https://www.availity.com" target="_blank">
```

Comment:
`<Link />` is from `@availity/link`, which does not have an `href` prop: https://github.com/Availity/availity-react/blob/master/packages/link/Link.js

This is a false positive picked up by that eslint rule. You are correct however, that it should be fixed. Rather than adding an `href` property, I believe importing the component as `AvLink` instead of `Link` will do the trick.

Reply:
Do we need to create a new PR or can this be fixed in next release?

Reply:
We can make the changes in this PR. We should just be able to copy what was done for `availity-starter-complex`: https://github.com/Availity/availity-starter-complex/blob/master/project/app/shared/Footer.js#L3 🚀